From 8dce3cac799bb873df0091c58a0473687e3c398d Mon Sep 17 00:00:00 2001
From: wenliang <wenliang@dhcpserver.shannon-data.com>
Date: Wed, 5 Sep 2018 21:09:39 +0800
Subject: [PATCH] patch test.

---
 patch | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 patch

diff --git a/patch b/patch
new file mode 100644
index 0000000..35bc9c2
--- /dev/null
+++ b/patch
@@ -0,0 +1 @@
+This is test for path.
-- 
1.8.3.1

